dbms/src/Functions/FunctionsVisitParam.h

```diff
@@ -87,61 +87,45 @@ struct ExtractBool
 
 struct ExtractRaw
 {
-    static void extract(const UInt8 * pos, const UInt8 * end, ColumnString::Chars_t & res_data)
+    inline static void skipAfterQuotationIfNeed(const UInt8 *& pos, const UInt8 * end, ColumnString::Chars_t & res_data)
```

Comment:
It would be more clear if we just put this code where it is called.
PS. I think, it should be named `skipAfterBackslashIfNeed`.